x-pack/plugins/apm/common/agent_name.ts

```diff
@@ -73,3 +74,7 @@ export function normalizeAgentName(agentName: string | undefined) {
 
   return agentName;
 }
+
+export function isIosAgent(agentName?: string) {
+  return agentName === 'iOS/swift';
```

Comment:
should this also be 'open-telemetry/swift' ?

Reply:
Not sure if we want to add that yet. Maybe we do. Any thoughts @AlexanderWert or @bryce-b?

Reply:
not sure whether some of the dimensions/fields make any sense with generic "open-telemetry/swift" as agent (such as device, mobile carrier, etc. ).

Reply:
maybe let's keep it with `iOS/swift` for now. @bryce-b WDYT?

Reply:
I've already added it. I can remove it again, or we can just leave it in and see what happens.

Reply:
👍 I would add it. It is possible that someone might use the Otel Swift sdk in a standalone state. Device, mobile carrier, etc will be included in the standard otel-swift sdk.